cmd/worker/clusterfeature.go

```diff
@@ -23,8 +23,13 @@ import (
 	clusterfeatureworkflow "github.com/banzaicloud/pipeline/internal/integratedservices/integratedserviceadapter/workflow"
 )
 
-func registerClusterFeatureWorkflows(worker worker.Worker, featureOperatorRegistry integratedservices.IntegratedServiceOperatorRegistry, featureRepository integratedservices.IntegratedServiceRepository) {
-	worker.RegisterWorkflowWithOptions(clusterfeatureworkflow.IntegratedServiceJobWorkflow, workflow.RegisterOptions{Name: clusterfeatureworkflow.IntegratedServiceJobWorkflowName})
+func registerClusterFeatureWorkflows(worker worker.Worker, featureOperatorRegistry integratedservices.IntegratedServiceOperatorRegistry, featureRepository integratedservices.IntegratedServiceRepository, isV2 bool) {
+	workflowFunc := clusterfeatureworkflow.IntegratedServiceJobWorkflow
+	if isV2 {
```

Comment:
Workflow functions could be registered with different names. It would only make sense if the workflows are started by different callers though. If not, I guess this is fine.

Reply:
Right now the caller is the same, but we will change this in a follow up PR because v1/v2 workflows should be available for the v1/v2 apis respectively when we add the router layer that can dynamically dispatch to both.